<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_logoUAT">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../recursos.qrc">:/logo/Imagenes/UAT-Escudo-2024.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_logoFIT">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../recursos.qrc">:/logo/Imagenes/LogoFIT.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_logoCT">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>400</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../recursos.qrc">:/logo/Imagenes/logoCT.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>430</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_COM">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>430</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>COM8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Accion">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>430</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Conectar</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_estado">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Desconectado</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>200</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Yu Gothic UI&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>270</y>
      <width>311</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>El sensor ultrasonico envia se√±ales</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_enviar">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>130</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Distancia</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>659</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../recursos.qrc"/>
  <include location="../recursos.qrc"/>
 </resources>
 <connections/>
</ui>
